integration-tests/src/test/scala/tech/beshu/ror/integration/suites/RorStartingSuite.scala

```diff
@@ -0,0 +1,189 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.integration.suites
+
+import cats.data.NonEmptyList
+import monix.eval.Task
+import monix.execution.Scheduler
+import monix.execution.atomic.AtomicInt
+import org.scalatest.wordspec.AnyWordSpec
+import tech.beshu.ror.utils.containers.EsContainerCreator.EsNodeSettings
+import tech.beshu.ror.utils.containers._
+import tech.beshu.ror.utils.containers.images.ReadonlyRestPlugin.Config.Attributes
+import tech.beshu.ror.utils.elasticsearch.BaseManager.JSON
+import tech.beshu.ror.utils.elasticsearch.{ClusterManager, SearchManager}
+import tech.beshu.ror.utils.gradle.RorPluginGradleProject
+import tech.beshu.ror.utils.httpclient.RestClient
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
+
+trait RorStartingSuite extends AnyWordSpec with EsContainerCreator {
+
+  import RorStartingSuite._
+
+  implicit val scheduler: Scheduler = Scheduler.computation(10)
+
+  private val validRorConfigFile = "/basic/readonlyrest.yml"
+  private val atomicInt: AtomicInt = AtomicInt(1)
+
+  private val notStartedResponseCodeKey = "readonlyrest.not_started_response_code"
+
+  "ES" when {
```

Comment:
this test costs us too much. But I understand the need to have such tests (here and in another parts of ROR). We should consider improving this scenario in the future,